src/sdk/tests/web3/test_factory.py

```diff
@@ -34,7 +53,7 @@ def test_w3_valid_provider():
     """Should return valid w3 wrapper with valid provider"""
 
     valid_chain = "rinkeby"
```

Comment:
I wonder if you should make these a mapping to the numerical chain ID, like rinkeby is 4